dawn/renderer/actions/test/EditorActions.test.js

```diff
@@ -0,0 +1,58 @@
+import { expect } from 'chai';
+import { openFile, saveFile, deleteFile, createNewFile, editorUpdate, changeTheme, increaseFontsize, decreaseFontsize } from '../EditorActions';
```

Comment:
Split this input into multiple lines to avoid running off the screen. See https://github.com/pioneers/PieCentral/blob/master/dawn/renderer/components/Editor.js#L4 for an example.